tools/godotcpp.py

```diff
@@ -183,6 +183,8 @@ def options(opts, env):
         )
     )
 
+    opts.Add(BoolVariable("disable_exceptions", "Force disabling exception handling code", True))
```

Comment:
```suggestion
    opts.Add(BoolVariable("disable_exceptions", "Force disabling exception handling code", default=env.get("disable_exceptions", True)))
```

Reply:
Ah, and that would match the other option defaults as well!

Reply:
Thanks, applied!